src/Carousel/widget/components/Glyphicon.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * Original code can be found at:
+ * https://github.com/react-bootstrap/react-bootstrap/blob/master/src/Glyphicon.js
+ * 
+ * This code is under the MIT license found here:
+ * https://github.com/react-bootstrap/react-bootstrap/blob/master/LICENSE
+ */
+import classNames = require("Carousel/lib/classnames");
+import * as React from "Carousel/lib/react";
+
+import { prefix } from "../utils/bootstrapUtils";
+
+export interface GlyphiconProps extends React.Props<Glyphicon> {
+    /**
+     * An icon name. See e.g. http://getbootstrap.com/components/#glyphicons
+     */
+    glyph?: string;
+    className?: string;
+    bootstrapClass?: string;
+    elementProps?: {};
+}
+
+class Glyphicon extends React.Component<GlyphiconProps, {}> {
+    public static defaultProps: GlyphiconProps = {
```

Comment:
Seems not so useful, don't think this is used anywhere else right? If not, remove it by inlining it.